templates/namespace.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    {{- if .Values.ttl }}
+    janitor/ttl: {{ .Values.ttl }}
+    {{- end }}
+    field.cattle.io/projectId: {{ .Values.rancherProjectId }}
+    git/branch: {{ .Values.gitBranch }}
+    git/remote: SocialGouv/standup
+  labels:
+    azure-pg-admin-user: standup
```

Comment:
should be configurable (and hopefully useless at some point)

Reply:
it's done